src/doc/trpl/concurrency.md

```diff
@@ -0,0 +1,391 @@
+% Concurrency
+
+Concurrency and parallelism are incredibly important topics in computer
+science, and are also a hot topic in industry today. Computers are gaining more
+and more cores, yet many programmers aren't prepared to fully utilize them.
+
+Rust's memory safety features also apply to its concurrency story too. Even
+concurrent Rust programs must be memory safe, having no data races. Rust's type
```

Comment:
This sentence seems somewhat messed up. Did you intend to say something like "to prevent data races"?